demo/src/style/demos.css

```diff
@@ -41,3 +41,17 @@ ngbd-table-filtering span.ngb-highlight {
 ngbd-table-complete span.ngb-highlight {
   background-color: yellow;
 }
+
+.picsum-img-wrapper {
```

Comment:
Some nitpicking here, but I would actually have prefixed that selector with `ngb-caroussel` so that we know it's used inside caroussel demos just by reading here.

```css
ngb-caroussel .picsum-img-wrapper {
}
```